test/test_ui.py

```diff
@@ -25,7 +25,8 @@
         news_rss_feed="https://www.reddit.com/r/touhou.rss",
         game_binary=dict(Windows="fc.exe", Linux="fc.x86_64"),
         launch_flags=dict(Windows=["-bill", "-gates"],
-                          Linux=["-linus", "-torvalds"]),
+                          Linux=["-linus", "-torvalds"],
+                          Darwin=["-linus", "-torvalds"]),
```

Comment:
Please add a comment here regarding why you added this. Including a link to https://en.wikipedia.org/wiki/Darwin_(operating_system), since it's uncommon to see Darwin as the system platform.